### PR TITLE
Update README - Add more badges, link to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by DJCrapsody**
Add badges. They make README look awesome, thanks shields.io!
Download link will always be /release/latest as both Online and Offline apks can be found there

Need feedback on download and changelog (version) badge. Will people feel like they are buttons that lead to release page or changelog page?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,20 +23,16 @@ GoIV is an android app made for Pokémon GO that allows users to seamlessly calc
 ![Alt text](https://i.imgur.com/xXr9zzK.jpg "Advanced Information in Results Popup")
 
 ## Download
-### Latest Version: 3.1.0 ([Changelog](CHANGELOG.md))
+[![Github All Releases](https://img.shields.io/github/downloads/farkam135/goiv/total.svg?maxAge=2592000)](https://github.com/farkam135/GoIV/releases/latest)
+[![GitHub release](https://img.shields.io/github/release/farkam135/goiv.svg?maxAge=2592000)](CHANGELOG.md)
```
Comment (Blaisorblade) [inline README.md]: 1. When I hover these links I don't see the alt-text, just the link.
2. I'd rather have labels before the images just to be sure—the first label should be "Latest version" or "Latest release", and the second should be "Changelog".

**Comment by Blaisorblade** [comment]
BTW: shields.io has also an alternative Travis badge with more uniform layout, shouldn't we use that too?

**Comment by DJCrapsody** [comment]
Added alt text should be easy. I wouldn't mind using travis-ci badge on shields.io instead.

**Comment by Blaisorblade** [comment]
I'm happy to merge once you do those changes then ;-).

**Comment by Eremiell** [comment]
Sure, go for it. I pretty much used first working travis badge I found atm. I'm all for standardization.

**Pull request merged**